hamilton/plugins/h_ray.py

```diff
@@ -209,20 +209,33 @@ class RayTaskExecutor(executors.TaskExecutor):
     This is still experimental, so the API might change.
     """
 
-    def __init__(self, num_cpus: int):
+    def __init__(
+        self,
+        num_cpus: int = None,
+        config: typing.Dict[str, typing.Any] = None,
```

Comment:
`ray_init_config` might be more descriptive

Reply:
done